assets/style.css

```diff
@@ -0,0 +1,251 @@
+a {
+  color: #08c !important;
+}
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: "Montserrat", sans-serif;
+}
+
+.container {
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.bg-black {
+  background-color: black;
+}
+
+.text-1 {
+  color: white;
+  font-weight: lighter;
+  font-size: 1em;
+}
+
+.hero-text-zero {
+  font-weight: lighter !important;
+  font-size: 4em;
+  margin-top: 1em;
+}
+
+.hero-text-one {
+  font-weight: lighter !important;
+  font-size: 2em;
+  margin-top: .5em;
+}
+
+.hero-text-two {
+  font-weight: lighter !important;
+  font-size: 1.3em;
+  margin-top: 1em;
+}
+
+.white-text {
+  color: white !important;
+  text-decoration: none;
+}
+
+.white-text:hover {
+  text-decoration: underline;
+}
+
+header {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  height: 534px;
+  background-image: url("hiking.jpg");
+  background-size: 1440px 1315px;
+  background-position-y: 534px;
+  width: 100%;
```

Comment:
- [ ] The selectors from line #58 to line #62 don't do any effect on your page so please remove them